components/notebook-dockerfiles/tensorflow/cpu.Dockerfile

```diff
@@ -0,0 +1,99 @@
+ARG BASE_IMAGE=763104351884.dkr.ecr.us-west-2.amazonaws.com/tensorflow-training:2.6.0-cpu-py38-ubuntu20.04
+
+FROM $BASE_IMAGE
+
+ARG NB_USER=jovyan
+
+# TODO: User should be refactored instead of hard coded jovyan
+USER root
+ENV DEBIAN_FRONTEND noninteractive
+ENV NB_USER $NB_USER
+ENV NB_UID 1000
+ENV HOME /home/$NB_USER
+ENV NB_PREFIX /
+
+
+# Use bash instead of sh
+SHELL ["/bin/bash", "-c"]
+
+RUN apt-get update && apt-get install -yq --no-install-recommends \
+  apt-transport-https \
+  build-essential \
+  bash \
+  bzip2 \
+  ca-certificates \
+  curl \
+  g++ \
+  git \
+  gnupg \
+  gnupg2 \
+  graphviz \
+  locales \
+  lsb-release \
+  openssh-client \
+  sudo \
+  unzip \
+  vim \
+  wget \
+  zip \
+  emacs \
+  python3-pip \
+  python3-dev \
+  python3-setuptools \
```

Comment:
[nit] Can we put all of these in alphabetically sorted order 

(same comment for other dockerfiles too)